Makefile

```diff
@@ -1,6 +1,11 @@
 .PHONY: all
 
 VERSION := 4.2.0
+RELEASE := 1
```

Comment:
curious if "?=" would be a better pattern in general than ":=", since it allows for the variables to be overridden by makefile command line arguments and inherited environment variables

Reply:
Thank you for the suggestion. I have changed the pattern used for both VERSION and RELEASE.